pymchelper/readers/fluka.py

```diff
@@ -58,17 +58,18 @@ def parse_usrbin(self, estimator):
                 page = Page(estimator=estimator)
                 page.title = detector.name
                 # USRBIN doesn't support differential binning type, only spatial binning is allowed
+
                 estimator.x = MeshAxis(n=detector.nx,
                                        min_val=detector.xlow,
                                        max_val=detector.xhigh,
-                                       name="Position (X)",
+                                       name="Radius (R)" if detector.type in (1, 11) else "Position (X)",
```

Comment:
Can you extract `(1,11)` to some variables with meaningful names ?
Is SHIELD-HIT12A using Radius (R) as well ?

Reply:
I will check this and Angle.
According to docs/inspect_converter.rst it uses `'Radius (R)', 'Angle (PHI)',`. I will check it on UI.

Reply:
`Angle (PHI)` seems reasonable as it clearly resembles azimuthal angle from spherical coordinate system.

https://en.wikipedia.org/wiki/Spherical_coordinate_system